opendbc/car/hyundai/interface.py

```diff
@@ -41,7 +41,7 @@ def _get_params(ret: structs.CarParams, candidate, fingerprint, car_fw, alpha_lo
         # this needs to be figured out for cars without an ADAS ECU
         ret.alphaLongitudinalAvailable = False
 
-      ret.enableBsm = 0x1e5 in fingerprint[CAN.ECAN]
+      ret.enableBsm = 0x1ba in fingerprint[CAN.ECAN]
```

Comment:
Generally, you'd want to match the CAN ID that carstate is using for BSM, in case certain platforms don't have anything going on with the BSM signals defined in carstate.